crates/bevy_ecs/src/entity/map_entities.rs

```diff
@@ -72,6 +72,21 @@ impl<T: MapEntities> MapEntities for Option<T> {
     }
 }
 
+impl<K: MapEntities + Eq + core::hash::Hash, V: MapEntities, S: BuildHasher + Default> MapEntities
```

Comment:
Let's import Hash for consistency.

```suggestion
impl<K: MapEntities + Eq + Hash, V: MapEntities, S: BuildHasher + Default> MapEntities
```